src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
+const DEFAULT_HEALTH = 100;
+const DAMAGE_VALUE = 50;
+
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = DEFAULT_HEALTH) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  removeDead() {
+    Animal.alive = Animal.alive.filter((item) => item.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name) {
```

Comment:
The order of parameters in the constructor of the `Herbivore` class should be `name` first and then `health`. This is to maintain consistency with the `Animal` class constructor.

Reply:
> The order of parameters in the constructor of the `Herbivore` class should be `name` first and then `health`. This is to maintain consistency with the `Animal` class constructor.

Good point